server/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java

```diff
@@ -729,6 +718,35 @@ private String retrieveValueToken(ValueIsTokenContext ctx) {
     return addTreeNode(ctx, ProcedureDivisionBodyNode::new);
   }
 
+  @Override
+  public List<Node> visitConstantName(ConstantNameContext ctx) {
+    String subroutineName = PreprocessorStringUtils.trimQuotes(ctx.getText()).toUpperCase();
+    return getLocality(ctx.getStart())
+        .map(
+            locality -> {
+              if (!subroutineService.getUri(subroutineName).isPresent()) {
+                reportSubroutineNotDefined(subroutineName, locality);
+              }
+              subroutineDefinitionMap.putIfAbsent(
+                  subroutineName,
+                  new SubroutineDefinition(
+                      getSubroutineLocation(
+                              new ImmutablePair<>(
+                                  subroutineName, subroutineService.getUri(subroutineName)))
+                          .stream()
+                          .findFirst()
+                          .orElse(
+                              new Location(PREF_IMPLICIT + IMPLICIT_SUBROUTINE_PATH, new Range())),
```

Comment:
It looks like we support predefined subroutines. Maybe, an empty string will work better here?

Reply:
Empty string does not help. We need it as explained by Anton.